TestApp/ios/Pods/AppCenter/AppCenter-SDK-Apple/iOS/AppCenterData.framework/Headers/MSServiceAbstract.h

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#import <Foundation/Foundation.h>
```

Comment:
I think this can be omitted, too

Reply:
This is pods from native SDK that we commit, you would need to patch Apple SDK, not react native SDK.